database/src/main/kotlin/no/nav/su/se/bakover/database/beregning/BeregningMapper.kt

```diff
@@ -83,18 +82,12 @@ internal data class PersistertMånedsberegning(
 }
 
 internal data class PersistertFradrag(
-    private val fradragstype: Fradragstype,
-    private val månedsbeløp: Double,
-    private val utenlandskInntekt: UtenlandskInntekt?,
+    override val fradragstype: Fradragstype,
+    override val månedsbeløp: Double,
+    override val utenlandskInntekt: UtenlandskInntekt?,
     override val periode: Periode,
-    private val tilhører: FradragTilhører
+    override val tilhører: FradragTilhører,
 ) : Fradrag {
-    override fun getFradragstype(): Fradragstype = fradragstype
-
-    @JsonAlias("totaltFradrag")
-    override fun getMånedsbeløp(): Double = månedsbeløp
```

Comment:
Den brukes ikke under det aliaset lenger? Var det noe som bare ble brukt av en gammel frontendversjon ellerno?

Reply:
Stemmer, @JsonAlias er kun for serialisering. Finnes ikke spor av den i produksjonsbasen, og heller ikke preprod, siden den er nylig wipet.